`wp acorn db` connection fails due to missing port with default `wordpress` config

### Version

v5.0.5

### What did you expect to happen?

I'm playing around with the Acorn CLI's various database commands, and I noticed an issue with the `wp acorn db` command to connect to the database CLI. This should normally open an interactive db shell.

I started with Acorn's default database.php config, which works fine for other commands.

### What actually happens?

The command fails with an `Empty value for 'port' specified` error.

I can make it work by adding `'port' => '3306'` to the `wordpress` connection config array.

I tested other commands that work fine and (presumably) use the Acorn-defined database connection: `migrate` (and subcommands), `db:table`, and `db:seed`. I'm not sure why this one behaves differently.

WordPress allows appending the port to `DB_HOST`, but that doesn't fix it here.

It seems like Acorn ought to include the port in the default config, and probably allow overriding it with a `DB_PORT` constant?

### Steps to reproduce

1. Set up a Bedrock site with Acorn and a mysql database configured properly in .env
2. Publish Acorn's default configs `wp acorn vendor:publish`
3. Run `wp acorn db`

### System info

macOS 15.7.3
Laravel Valet 4.11.0
mysql Ver 9.6.0 for macos14.8 on x86_64 (Homebrew)
PHP v8.4.17
WordPress v6.9.1
WP-CLI v2.12.0

### Log output

```shell
mysql: [ERROR] mysql: Empty value for 'port' specified.

In Process.php line 280:

  The command "'mysql' '--host=localhost' '--port=' '--user=root' '--default-character-s
  et=utf8mb4' 'wp_etch-bedrock'" failed.

  Exit Code: 5(Unknown error)
```

### Please confirm this isn't a support request.

Yes

--- a/config/database.php
+++ b/config/database.php
@@ -46,6 +46,7 @@ return [
         'wordpress' => [
             'driver' => 'mysql',
             'host' => DB_HOST,
+            'port' => env('DB_PORT', 3306),
             'database' => DB_NAME,
             'username' => DB_USER,
             'password' => DB_PASSWORD,

